arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/CreateSection.java

```diff
@@ -64,17 +65,23 @@ public class CreateSection extends TraceSection
 
   private final AccountFragment.AccountFragmentFactory accountFragmentFactory;
 
-  // Just before create
-  private AccountSnapshot preOpcodeCreatorSnapshot;
-  private AccountSnapshot preOpcodeCreateeSnapshot;
+  // First account rows
+  private AccountSnapshot firstCreator;
+  private AccountSnapshot firstCreatee;
+  private AccountSnapshot firstCreatorNew;
+  private AccountSnapshot firstCreateeNew;
 
-  // Just at the entry child frame
-  private AccountSnapshot childContextEntryCreatorSnapshot;
-  private AccountSnapshot childContextEntryCreateeSnapshot;
+  // Second account rows
+  private AccountSnapshot secondCreator;
+  private AccountSnapshot secondCreatee;
+  private AccountSnapshot secondCreatorNew;
+  private AccountSnapshot secondCreateeNew;
 
-  // Just at the entry child frame
-  private AccountSnapshot reEntryCreatorSnapshot;
-  private AccountSnapshot reEntryCreateeSnapshot;
+  // Second account rows
+  private AccountSnapshot thirdCreator;
+  private AccountSnapshot thirdCreatee;
+  private AccountSnapshot thirdCreatorNew;
+  private AccountSnapshot thirdCreateeNew;
```

Comment:
Just a big refactoring of `CreateSection`. The goal being to be close to the spec.